packaging/src/shopify-cli.rb.liquid

```diff
@@ -39,7 +39,7 @@ class ShopifyCli{% if formulaVersion %}AT{{formulaVersion}}{% endif %} < Formula
     executable_path = "#{libexec}/bin/shopify"
     FileUtils.move(executable_path, "#{executable_path}-original")
     executable_content = <<~SCRIPT
-      #!/usr/bin/env node
+      #!/usr/bin/env #{libexec}/bin/node
```

Comment:
hey @amcaplan should this be `#!#{libexec}/bin/node`? if we hard code the absolute path we shouldn't be needing `env` anymore